.gitignore

```diff
@@ -29,7 +30,7 @@
 .packages
 .pub-cache/
 .pub/
-build/
+/build/
```

Comment:
`build/` should cover `/build/`. 
Was this from `flutter migrate`?

Reply:
When you create a new Flutter project, these files generate automatically, including the .gitignore file.

Reply:
This actually has to be `build/` because of nested projects.

Reply:
The templates have trailing newlines.